kagenti/installer/app/components/ui.py

```diff
@@ -21,6 +21,27 @@
 
 def install():
     """Installs the Kagent UI from its deployment YAML."""
+    # Create the auth secret, containing the Keycloak client secret
+    run_command(
+        [
+            "kubectl",
+            "apply",
+            "-f",
+            str(config.RESOURCES_DIR / "auth-secret.yaml"),
+        ],
+        "Creating auth secret",
+    )
+    run_command(
+        [
+            "kubectl",
+            "wait",
+            "--for=condition=complete",
+            "job/auth-secret",
+            "-n", "kagenti",
+            "--timeout=300s",
+        ],
+        "Waiting for auth secret job to complete",
+    )
```

Comment:
This starts a job and also creates a service account, a role, and a role binding.

The purpose of the job is to create a `kagenti` client within Keycloak, extract its client secret, and put the client secret into a Kubernetes secret named `auth.

The service account, role, and role binding are required because the job is running in the `kagenti` namespace so it can access `environments` configmap but needs to create the `auth` secret in the `kagenti-system` namespace.